src/lib/Components/HoursCollapsible.tsx

```diff
@@ -0,0 +1,91 @@
+import { Box, Flex, Sans, Serif } from "@artsy/palette"
```

Comment:
@kierangillen did you mean to add that to this PR?

Reply:
Thanks, no, just removed it.